pages/index.tsx

```diff
@@ -82,11 +83,21 @@ const DocsPage = () => (
           </pre>
         </Card>
         <Heading variant="headline">Buttons</Heading>
-        {Object.keys(theme.buttons).map((key) => (
-          <Button key={key} variant={key} sx={{ mr: 3, mb: 3 }}>
-            {key} btn
-          </Button>
-        ))}
+        <Flex sx={{ flexWrap: 'wrap' }}>
+          {Object.keys(theme.buttons).map(key => (
+            <Box>
+              <Button key={key} variant={key} sx={{ mr: 3, mb: 3 }}>
+                {key} btn
+              </Button>
+            </Box>
+          ))}
+          <Box>
+            <Button sx={{ mr: 3, mb: 3 }}>
+              Hello
+              <HiArrowRight />
+            </Button>
+          </Box>
```

Comment:
```suggestion
          {Object.keys(theme.buttons).map(key => (
            <Flex sx={{flexDirection: 'column', gap: 3, mb: 3}} key={key}>
              <Button variant={key} sx={{ mr: 3}}>
                {key} btn
              </Button>
            <Button variant={key} sx={{ mr: 3}}>
              {key} btn
              <HiArrowRight />
            </Button>
          </Flex>
          ))}
          
```

We should test using icons for every variant, in my opinion.